ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationEditPage.java

```diff
@@ -32,30 +49,11 @@ public boolean validate() {
 		return getRootElement().is(visible);
 	}
 
-	public ActionConfigureComponent actionConfigureComponent() {
-		return new ActionConfigureComponent();
-	}
-
-	public FlowViewComponent flowViewComponent() {
-		return new FlowViewComponent();
-	}
-
-	public ConnectionSelectComponent connectionSelectComponent() {
-		return new ConnectionSelectComponent();
-	}
-
-	public IntegrationBasicsComponent basicsComponent() {
-		return new IntegrationBasicsComponent();
-	}
-
 	public ElementsCollection getAllTrashes() {
```

Comment:
So we probably do not need these methods here anymore.

Reply:
@lvydra correct